src/services/DFGVariable.jl

```diff
@@ -129,6 +131,48 @@ Interface function to return the manifold point type of an InferenceVariable, ex
 function getPointType end
 getPointType(::T) where {T <: InferenceVariable} = getPointType(T)
 
+"""
+    $SIGNATURES
+Interface function to return the user provided identity point for this InferenceVariable manifold, extend this function for all Types<:InferenceVariable.
+"""
+function getPointIdentity end
+getPointIdentity(::T) where {T <: InferenceVariable} = getPointIdentity(T)
+
+
+"""
+    $SIGNATURES
+
+Default escalzation from coordinates to a group representation point.  Override if defaults are not correct.
+E.g. coords -> se(2) -> SE(2).
+
+Related
+
+[`getCoordinates`](@ref)
+"""
+function getPoint(::Type{T}, v::AbstractVector) where {T <: InferenceVariable}
+    M = getManifold(T)
+    p0 = getPointIdentity(T)
+    X = get_vector(M, p0, v, DefaultOrthonormalBasis())
+    exp(M, p0, X)
+end
+
+"""
+    $SIGNATURES
+
+Default reduction of a variable point value (a group element) into coordinates as `Vector`.  Override if defaults are not correct.
+
+Related
+
+[`getPoint`](@ref)
+"""
+function getCoordinates(::Type{T}, p) where {T <: InferenceVariable}
+    M = getManifold(T)
+    p0 = getPointIdentity(T)
+    X = log(M, p0, p)
+    get_coordinates(M, p0, X, DefaultOrthonormalBasis())
+end
```

Comment:
I'm not sure about this. It's not general and makes a lot of assumptions. 

Perhaps it can be more accurate with a new abstract `LieGroupInferenceVariable <: InferenceVariable`  (but also don't like it that much)

Reply:
Lets not pre-optimize.  I'm trying to take a reasonable step towards Manifolds based on where we are today.  `Manifolds.get_coordinates` already exists, so my feeling is we can leverage that as default and user must override for general cases.

For this step, I'm totally against building new abstracts.  Maybe later once `vnd.val::Vector{P}` is already working fully downstream.